### ISSUE TITLE
Confirm box on logout

### ISSUE THREAD

**Issue by johnbabu021** [Planet-NULLCAST/curriculum#884]
while user clicks logout pop a confirm button 

create a confirm box

**Comment by thealpha93** [comment]
Would you like to work on this issue?? @johnbabu021

**Comment by johnbabu021** [comment]
yeah i can work on the issue but currently i am on a tight schedule. I can make it possible after 1 or 2weeks

**Comment by thealpha93** [comment]
Nice. I’ll assign it to you @johnbabu021 
Make sure you make it a reusable component. We’ll need it in other places too. Check if similar component already exists in the code base

**Comment by johnbabu021** [comment]
i am starting on this issue will make a pr in 2-3 days

**Comment by johnbabu021** [comment]
Login trouble in Development phase
(Request failed with 404).
.Working on nullcast.dev

**Comment by thealpha93** [comment]
Cool. Connect with me if you run into any issue
